preflight-checks/src/index.js

```diff
@@ -115,7 +116,7 @@ const result = checks.reduce( async ( priorCheck, check, index ) => {
 	} );
 }, Promise.resolve() );
 
-result.then( () => {
+result.then( async () => {
```

Comment:
It's generally an antipattern to mix Promise.then and async/await, but no big deal since this package will go away sometime soon